### PR TITLE
[CORE-10707] Refactor TextFieldMeta component to support FieldGroup

### PR DESCRIPTION
**Pull request by iamuchejude**
Relates to https://github.com/good-hood-gmbh/nebenan-frontend/pull/1718

### CODE REVIEW

File: packages/inputs-minimal/src/textfields_meta/index.jsx [hunk 1]
```diff
@@ -6,19 +6,22 @@ const Meta = ({
   error,
   hint,
   className,
-}) => (
-  <div className={clsx(styles.meta, className)}>
-    {error && (
+}) => {
+  const hasHintOrError = Boolean(hint || error);
+  const useInputGroup = hasHintOrError && typeof error !== 'string';
```
Comment (peterhass) [inline packages/inputs-minimal/src/textfields_meta/index.jsx:11]: `input-minmal` has no idea what an 'input group' is. It doesn't exist in the package. Maybe just go with ...

```js
const hasError = typeof error === 'string';
const hasHint = Boolean(hint);

if (!hasError && !hasHint) return null;
```

File: packages/inputs-minimal/src/textfields_meta/index.module.scss [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 .meta {
   display: flex;
+  font-weight: $font-weight-l;
```
Comment (peterhass) [inline packages/inputs-minimal/src/textfields_meta/index.module.scss:5]: is this in the design system?

**Review by peterhass — CHANGES_REQUESTED** [review]

File: packages/inputs-minimal/src/textfields_meta/index.jsx (hunk 1 above)
Comment (iamuchejude) [reply packages/inputs-minimal/src/textfields_meta/index.jsx:11]: Yeah, that makes sense.

File: packages/inputs-minimal/src/textfields_meta/index.module.scss (hunk 2 above)
Comment (iamuchejude) [reply packages/inputs-minimal/src/textfields_meta/index.module.scss:5]: Yeah, it is.

**Review by peterhass — APPROVED** [review]

**Comment by peterhass** [comment]
Will merge this PR because you already published a stable version anyway (and we're publishing-blocked otherwise)

**Pull request merged**